Tests/TwoDimension/AreaTests.cs

```diff
@@ -170,15 +170,66 @@ public void PositionSet()
 		[Test]
 		public void PositionGet()
 		{
-			// TODO: Issue 6 (https://github.com/Wizcorp/TileSystem/issues/6)
-			Assert.Fail();
-		}
+            Area area = new Area();
+            var mockLevel = new Mock<ILevel>();
```

Comment:
Is this necessary? I don't think to perform this test you need to use the `SetPosition(level, pos)` on the area. This is used when we have multiple areas, and do not think it should affect any methods that are looking inside the area directly such as get neighbours.

Reply:
Indeed, this is not useful for this test, and neither for PositionGetNeighbours test.